api/forms.py

```diff
@@ -26,16 +27,28 @@ def contact_submit():
     This function handles the contact submissions.
     """
     data = request.json
-    name = data['name']
-    email = data['email']
-    message = data['message']
 
-    # TODO: here we will put the message in the database
-    # as unseen.
-    print(f'Name: {name}, email: {email}')
-    print(f'Message: {message}')
+    if not (data['name'] and data['email'] and data['message']):
+
+        alert('Failed')
```

Comment:
Let's return an error status (Ex. statusCode = 400) to the frontend and handle this with the modal on the frontend